src/Service/Modules/PurchaseOrderModule.cs

```diff
@@ -116,6 +117,18 @@ private async Task GetApp(HttpRequest req, HttpResponse res)
             await res.WriteAsync(result);
         }
 
+        private async Task EmailOrderPdf(
+            HttpRequest req,
+            HttpResponse res,
+            int orderNumber,
+            string emailAddress,
```

Comment:
not important right now but will probably need multiple email address option and cc to self etc